pubsub/README.rst

```diff
@@ -98,7 +98,8 @@ the topic, and subscribe to that.
         project_id=os.getenv('GOOGLE_CLOUD_PROJECT'),
         sub='MY_SUBSCRIPTION_NAME',  # Set this to something appropriate.
     )
-    subscriber.create_subscription(name=subscription_name, topic=topic_name)
```

Comment:
Hello @tseaver. I don't understand why we should split this line? It has 72 characters. I checked google style guide and maximum line length is 80. Just I want to understand the reason. :)